modules/nw-networkpolicy-multitenant-isolation.adoc

```diff
@@ -70,7 +65,7 @@ spec:
   - from:
     - namespaceSelector:
         matchLabels:
-          network.openshift.io/policy-group: monitoring
+          policy-group.network.openshift.io/monitoring: ""
```

Comment:
I don't think we actually got around to updating the monitoring namespace to use the new label, so this actually won't work.

Probably best to just keep using the old label syntax (`network.openshift.io/policy-group: ...`) everywhere for now.
(Other than for `policy-group.network.openshift.io/host-network: ""`, where there is no corresponding "old-style" label. But you don't talk about that in this document so that doesn't matter.)

Reply:
@danwinship, sure, that makes sense. Thanks! (I've created a new Jira to track this, so if this ever changes, we can update everywhere to the newer labels.)